data_source_transip_private_network.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/transip/gotransip/v6/repository"
+	"github.com/transip/gotransip/v6/vps"
+)
+
+func dataSourcePrivateNetwork() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourcePrivateNetworkRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Required: true,
+				Type:     schema.TypeString,
+				ForceNew: true,
+			},
+			"description": {
+				Computed: true,
+				Type:     schema.TypeString,
+			},
+			"is_blocked": {
+				Computed: true,
+				Type:     schema.TypeString,
+			},
+			"is_locked": {
+				Computed: true,
+				Type:     schema.TypeString,
+			},
+			"vps_names": {
+				Computed: true,
+				Type:     schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+		},
+	}
+}
+
+func dataSourcePrivateNetworkRead(d *schema.ResourceData, m interface{}) error {
+	client := m.(repository.Client)
+	repository := vps.PrivateNetworkRepository{Client: client}
+	name := d.Get("name").(string)
+	// getPrivateNetworkID(d, m)
```

Comment:
Commented out code can be removed if it's not used.

Reply:
I agree, removed above comment and the unused function it refers to.